cli-plugins/examples/badmeta/main.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
I don't think this "example" adds much value, it should be an e2e test instead. My 2cts is that we should only have valid examples users can copy/paste to bootstrap their plugins.

Reply:
Actually this is only here to support the e2e testing, so you are right it's not really an example, I was just hoping to not need to battle the build system a second time to get it built.

So I'll move this one to e2e/cli-plugins/test-plugins and arrange (*somehow*) to have it built.